### PR TITLE
[Delivers #78740108] add Google Analytics

### PR DESCRIPTION
**Pull request by afeld**
Followed instructions at https://github.com/18F/analytics-standards. Not clear to me if it's being filtered out for all domains but https://cap.18f.gov/ – I wasn't seeing any [realtime traffic](https://www.google.com/analytics/web/#realtime/rt-overview/a48605964w99418896p103376476/) – so not sure if we need to exclude it in other environments (see https://github.com/18F/analytics-standards/issues/3).

@gbinal @ramirezg Mind taking a quick look and making sure that I did this right?

### CODE REVIEW

**Comment by afeld** [comment]
jk!

screen shot 2015-06-08 at 8 37 48 pm

File: app/views/layouts/_analytics.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
+<!-- 18F analytics -->
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-48605964-27', 'auto');
```
Comment (cmc333333) [inline app/views/layouts/_analytics.html.erb]: Probably a good idea to set this as an environmental variable/setting, if only to allow us swap it out in different environments.

**Comment by afeld** [comment]
Updated!

**Comment by afeld** [comment]
Got thumbs up from @cmc333333 and @konklone [in Slack](https://18f.slack.com/archives/g-analytics/p1433880221000143).

**Pull request merged**